.github/ISSUE_TEMPLATE.md

```diff
@@ -1,27 +1,36 @@
+
+## READ ME FIRST
+
+This repository is for reporting issues related to Docker Documentation. Before submitting a new issue, check whether the issue has already been reported. You can join the discussion using an emoji, or by adding a comment to an existing issue.
+
+You can ask general questions and get community support through the Docker Community Slack - http://dockr.ly/slack. Personalized support is available through the Docker Pro, Team, and Business subscriptions. See https://www.docker.com/pricing for details.
+
```

Comment:
Do we want this part to be visible in the issue that's opened, or only visible to the person opening the ticket? If the latter, we could put this section as a HTML comment;

```
<!--
## READ ME FIRST

.....

-->
```

Reply:
I think it helps if it's visible to the person opening the ticket. Let me update

Reply:
Updated